account_mtd/controllers/main.py

```diff
@@ -3,9 +3,29 @@
 from odoo import http, exceptions
 from datetime import datetime, timedelta
 from werkzeug.utils import redirect
+from odoo.http import request
```

Comment:
_logger should be after imports and I don't think you need to import requests separately because you've already imported http.

```
from datetime import datetime, timedelta
from werkzeug.utils import redirect

from odoo import http, exceptions

_logger = logging.getLogger(__name__)
```